packages/ui/src/components/ChatOptionsSheet.tsx

```diff
@@ -635,6 +639,43 @@ export function ChannelOptions({
             } as ActionGroup,
           ]
         : []),
+      ...(group &&
+      !['groupDm', 'dm'].includes(channel.type) &&
```

Comment:
This logic looks right, we might wanna define this in an easier to read way elsewhere, though.

Reply:
"canInvite," I guess. I'll make a note and follow up later.